### DISCUSSION TITLE
NPCs currently don't use the absorb action after digesting someone, meaning that they keep their giant belly. Could this be changed?

### DISCUSSION THREAD

**Discussion by Superjosh58** [Squidpad/BG3DevouringAndDigesting#35] [Q&A]
As above. NPCs don't use it and they keep their big bellies for rather a while. Could they start using this themselves or is there a good reason why they don't (eg, stealing quest items)?

**Comment by AraHaan** [comment]
They could then take your entire inventory this way with no way to take it back without having a mod to barter with any npc, or using ``Die`` & ``Resurrect`` with them.
